TheraLang.BLL/Services/UserManagementService.cs

```diff
@@ -28,14 +28,21 @@ public class UserManagementService : IUserManagementService
 
         public async Task<User> GetUser(string email, string password)
         {
-            var user = await _unitOfWork.Repository<User>().GetAll().Include(x => x.Role)
-                .FirstOrDefaultAsync(u => u.Email == email);
-            if (PasswordHasher.VerifyHashedPassword(user.PasswordHash, password))
+            try
             {
-                return user;
-            }
+                var user = await _unitOfWork.Repository<User>().GetAll().Include(x => x.Role)
+                    .FirstOrDefaultAsync(u => u.Email == email);
+                if (PasswordHasher.VerifyHashedPassword(user.PasswordHash, password))
+                {
+                    return user;
+                }
 
-            throw new Exception($"Cannot get user with {nameof(email)}: {email}.");
+                throw new Exception($"Cannot get user with {nameof(email)}: {email}.");
```

Comment:
Nu chto eto?
We should use ApiException instead of Exception in handled situations.

Reply:
done